Integreat/Integreat.Shared/ViewModels/MainContentPageViewModel.cs

```diff
@@ -75,14 +74,20 @@ public class MainContentPageViewModel : BaseContentViewModel
             ItemTappedCommand = new Command(OnPageTapped);
             OpenSearchCommand = new Command(OnOpenSearch);
             ChangeLanguageCommand = new Command(OnChangeLanguage);
-            ChangeLocationCommand = new Command(OnChangeLocation);
             OpenContactsCommand = new Command(OnOpenContacts);
 
+            ShowHeadline = Device.RuntimePlatform != Device.Android;
 
-            // add search icon to toolbar
+            // add toolbar items
             ToolbarItems = new List<ToolbarItem>
             {
-                new ToolbarItem { Text = AppResources.Search, Icon = "search", Command = OpenSearchCommand},
+                new ToolbarItem { Text = AppResources.Search, Icon = "search", Order = ToolbarItemOrder.Primary, Command = OpenSearchCommand},
+                new ToolbarItem { Text = AppResources.Language, Icon = "translate", Order = ToolbarItemOrder.Primary, Command = ChangeLanguageCommand },
+#if __ANDROID__
+                new ToolbarItem { Text = AppResources.Share, Order = ToolbarItemOrder.Secondary, Icon = "share", Command = ContentContainerViewModel.Current.ShareCommand },
+                new ToolbarItem { Text = AppResources.Location, Order = ToolbarItemOrder.Secondary, Command = ContentContainerViewModel.Current.OpenLocationSelectionCommand },
```

Comment:
Duplicate in ExtrasContentPageViewModel

Reply:
Yes there are duplications in all the BaseContentVieModels but it's the only way to get is work in iOS